### ISSUE TITLE
System.AggregateException when ran

### ISSUE THREAD

**Issue by AronParker** [BaseMC/Aves#22]
Tested with Minecraft versions 1.15.2, 20w11a and 20w12a, getting the same exception for each and every one of them:

```14:18:41,604 WRN 6  ProcessUtil [RunProcess] 912 finished with ExitCode=-1
14:18:41,617 ERR 1  Program [Run] System.AggregateException: One or more errors occurred. (One or more errors occurred. (A task was canceled.))
 ---> System.AggregateException: One or more errors occurred. (A task was canceled.)
 ---> System.Threading.Tasks.TaskCanceledException: A task was canceled.
   at Aves.MakeRead.Provider.MRDeobfProvider.RunDeobfuscation(Configuration config, VariantConfig variant)
   at Aves.MakeRead.Provider.MRDeobfProvider.<>c__DisplayClass1_0.<Run>b__0()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at Aves.MakeRead.Provider.MRDeobfProvider.Run(Configuration config, VariantConfig variantConfig)
   at Aves.MakeRead.MakeReadableVariant.Run()
   at Aves.MakeRead.MakeReadable.<>c__DisplayClass6_0.<Run>b__1()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.WaitAllCore(Task[] tasks, Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at Aves.MakeRead.MakeReadable.Run()
   at Aves.Runner.Run()
   at Aves.StartUp.DoStart()
   at Aves.StartUp.Start()
   at Aves.Program.<>c.<Run>b__1_0(CmdOption opt)
   at CommandLine.ParserResultExtensions.WithParsed[T](ParserResult`1 result, Action`1 action)
   at Aves.Program.Run(String[] args)
14:18:41,630 INF 2  LoggerInitializer [InitLogger] Shutting down logger; Flushing...```

**Comment by litetex** [comment]
Can you post the full log output? 
You can also generate a log file with the option ``-l``. The file is generated into the ``logs`` folder of the current directory.

Additional information that would be useful:
* What version are you using?
* With which parameters did you start the program?

**Comment by AronParker** [comment]
> Can you post the full log output?
> You can also generate a log file with the option `-l`. The file is generated into the `logs` folder of the current directory.
> 
> Additional information that would be useful:
> 
> * What version are you using?
> * With which parameters did you start the program?

I am using Aves 1.0.7366.36619
I used `.\Aves.exe -v 20w12a -l` in this example

The log file is here if that helps: https://pastebin.com/4ZU1nC9w

**Comment by litetex** [comment]
Wow that was quick 👍 
Found the error here:
``
19:25:41,113 INFO  14 ProcessUtil [RunProcess] 9104 >> Caused by: java.lang.ArithmeticException: / by zero
19:25:41,114 INFO  14 ProcessUtil [RunProcess] 9104 >>  at java.lang.Math.floorDiv(Unknown Source) ~[?:?]
19:25:41,117 INFO  12 ProcessUtil [RunProcess] 9104 >>  at javgent.executor.execmodules.Executor.unpackJarInternal(Executor.java:278) 
``

Fixing is in progress

**Comment by litetex** [comment]
Please try replacing your ``javgent.jar`` with this one:
[javgent-standalone.zip](https://github.com/BaseMC/Aves/files/4377168/javgent-standalone.zip)
(Built from https://github.com/BaseMC/javgent/commit/d642436e08622e88a0c525dc0f8a63cdef1d4a97)

and inform me, if the error still exists

**Comment by AronParker** [comment]
Thanks for the quick fix! Works fine now 👍

**Comment by litetex** [comment]
Should be fixed in [v0.2.2](https://github.com/BaseMC/Aves/releases/tag/v0.2.2)
